uploader.v2.json

```diff
@@ -1372,7 +1372,7 @@
             "symbolic_name": "ceph_conf"
         },
         {
-            "file": "/var/log/ceph/ceph.log",
+            "file": "/var/log/ceph/()ceph.log*",
```

Comment:
@psachin this should have been converted to a glob spec and moved to the `globs` section for `uploader.v2.json`.  I realize it may be confusing, but this version of the json is for the most recent clients.

Reply:
@bfahr Moved both `ceph_osd_log` & `ceph_log` to globs section.